### PR TITLE
MUZIMA-271- Changes in Teleconsultation form to enable validation and…

### PR DESCRIPTION
**Pull request by ManishaRana1195**
… autocompletion on location and provider fields

### CODE REVIEW

File: js/muzima.js [hunk 1]
```diff
@@ -820,17 +830,27 @@ $(document).ready(function () {
...
                 validProviderOnly: true
             }
         });
         /* End - validProviderOnly*/
-    }
 
-    document.setupValidationForLocation = function(value, element, listOfLocations) {
+
+    document.setupValidationForLocation = function(value, element) {
+             var listOfLocations = [];
```
Comment (nribeka) [inline js/muzima.js]: Why do we add two listOfLocations here?

File: js/muzima.js [hunk 2]
```diff
@@ -728,6 +728,19 @@ $(document).ready(function () {
 
     /* End - Code to Serialize form along with Data-Concepts */
 
+     document.setupAutoCompleteData = function(elementName) {
+         var dataDictionary = [];
```
Comment (nribeka) [inline js/muzima.js]: Why are we adding the variable twice here?

Comment (ManishaRana1195) [inline js/muzima.js]: Hi @nribeka , 
  Its not required to add the variable twice. I will make the changes and generate a new pull request.

File: js/muzima.js (hunk 1 above)
Comment (ManishaRana1195) [inline js/muzima.js]: I will remove it and check-in the changes.

**Pull request merged**